.gitignore

```diff
@@ -21,7 +21,7 @@
 /test/**/*.xml
 /test/dummy.cpp
 /test/**/*.so
-
+**/math/**/*
```

Comment:
Doesn't this ignore any math directories anywhere? I also don't think we want to ignore the math submodule in general. What was this aimed at?

Reply:
Oh I thought we would ignore the math repo and users would have it locally. I can remove this